src/annotator/test/adder-test.js

```diff
@@ -217,6 +219,95 @@ describe('Adder', () => {
     });
   });
 
+  describe('#_findZindex', () => {
```

Comment:
Can you update the description here since these tests are no longer calling this private method. Something like `describe('adder Z index', ...` would be fine.